EmailValidationProcessOrchestrator/__tests__/handler.test.ts

```diff
@@ -17,74 +17,102 @@ import {
   ActivityResult as SendValidationEmailActivityResult
 } from "../../SendValidationEmailActivity/handler";
 import {
-  handler,
+  getHandler,
   OrchestratorInput as EmailValidationProcessOrchestratorInput
 } from "../handler";
+import { FeatureFlagEnum } from "../../utils/featureFlag";
 
 describe("EmailValidationProcessOrchestrator", () => {
-  it("should start the activities with the right inputs", async () => {
-    const emailValidationProcessOrchestratorInput = EmailValidationProcessOrchestratorInput.encode(
-      {
-        email: aEmail,
-        fiscalCode: aFiscalCode
-      }
-    );
+  it.each([
+    {
+      betaList: [],
+      ff: FeatureFlagEnum.NONE,
+      expectedEamilValidationActivity: "SendValidationEmailActivity"
+    },
+    {
+      betaList: [aFiscalCode],
+      ff: FeatureFlagEnum.NONE,
+      expectedEamilValidationActivity: "SendValidationEmailActivity"
+    },
+    {
+      betaList: [aFiscalCode],
+      ff: FeatureFlagEnum.BETA,
+      expectedEamilValidationActivity: "SendTemplatedValidationEmailActivity"
+    },
```

Comment:
We could add another test to check routing in case of non-beta user with FF=BETA
```typescript
    {
      betaList: [anotherFiscalCode],
      ff: FeatureFlagEnum.BETA,
      expectedEamilValidationActivity: "SendValidationEmailActivity"
    },
 ```

Reply:
moved to UpsertedProfileOrchestrator tests